admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -89,6 +90,7 @@ module.exports = {
     order_states: './js/pages/order-states',
     order_return_states_form: './js/pages/order-return-states/form',
     order_view: './js/pages/order/view.js',
+    outstanding: './js/pages/outstanding',
```

Comment:
Entry point `'./js/pages/outstanding'` is used twice, one for `customer_outstanding` and one for `outstanding`. Only one should be used